### PR TITLE
Update docker. Clean up chart

### PR DESCRIPTION
**Pull request by luigibk**
### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: charts/ccpay-bubble-frontend/values.yaml [hunk 1]
```diff
@@ -1,11 +1,26 @@
...
   applicationPort: 3000
+  ingressHost: ccpay-bubble.{{ .Values.global.environment }}.platform.hmcts.net
+  aadIdentityName: ccpay
+  memoryLimits: 1024Mi
+  cpuLimits: 1000m
+  readinessDelay: 30
+  livenessDelay: 30
+  keyVaults:
+    "ccpay":
+      resourceGroup: ccpay
```
Comment (timja) [inline charts/ccpay-bubble-frontend/values.yaml:11]: this isn't needed
```suggestion
```

Comment (luigibk) [reply charts/ccpay-bubble-frontend/values.yaml:11]: Ok, I'll remove at the next PR as I have something else to clean up.

File: start.js [hunk 2]
```diff
@@ -10,23 +11,21 @@ appInsights.setAuthenticatedUserContext = userId => {
...
-process.env.NODE_TLS_REJECT_UNAUTHORIZED = '1';
-
-// SSL handled at IIS level so Node.js app (iisnode) should be http only.
-if (process.env.IGNORE_CERTS) {
-  http.createServer(app).listen(port);
-} else {
+// reverse proxy handles tls in non local environments
+if (process.env.NODE_ENV === 'development') {
+  // Disable cert errors
+  process.env.NODE_TLS_REJECT_UNAUTHORIZED = '1';
```
Comment (timja) [inline start.js:23]: this shouldn't be here

Comment (luigibk) [reply start.js:23]: I agree

**Pull request merged**